src/core/thread/network_data_leader_ftd.cpp

```diff
@@ -740,11 +794,10 @@ otError Leader::AddBorderRouter(PrefixTlv &aPrefix, BorderRouterTlv &aBorderRout
         dstContext->SetContextLength(aPrefix.GetPrefixLength());
     }
 
-    VerifyOrExit(dstContext != NULL, error = OT_ERROR_NO_BUFS);
+    dstContext->SetCompress();
```

Comment:
This is new change, right?

Reply:
This should maintain the existing behavior.  The Compress Flag is set if we have a valid Context.

Reply:
Ok. Thanks.